src/coordinator.coffee

```diff
@@ -486,6 +486,17 @@ class Coordinator extends EventEmitter
           port: parts[3]
       graph.connections.push edge
 
+    iipIds = Object.keys(@iips).sort()
```

Comment:
Would be a little bit cleaner to sort aftewards by tgt.process/port, but I guess this is effectively the same thing.